src/Serilog.Sinks.Datadog.Logs/Sinks/Datadog/DatadogHttpClient.cs

```diff
@@ -77,6 +77,8 @@ private async Task Post(JsonPayloadBuilder payloadBuilder)
         {
             var payload = payloadBuilder.Build();
             var content = new StringContent(payload, Encoding.UTF8, _content);
+            HttpResponseMessage lastResponse = null;
```

Comment:
`lastResponse` is never assigned. I think this should be assigned on or after line `92`.

Reply:
Yeah, bad copy and paste.